components/ModSubmissionListContainer.vue

```diff
@@ -16,41 +16,73 @@
             {{ $t("modPanelSubmissionList.submitted") }}
         </div>
 
-        <div v-for="(fakeDatum, index) in modScreenStore.fakeData" :key="index" class="grid grid-cols-subgrid col-span-5 bg-gray-200" >
-            <button @click="handleClickToSubmissionForm(fakeDatum.id)"
-            :data-testid='`mod-submission-list-item-${index + 1}`'
+        <div v-if="hasSubmissions" v-for="(submission, index) in modSubmissionsStore.submissionsData" :key="index"
             class="grid grid-cols-subgrid col-span-5 bg-gray-200">
+            <div @click="handleClickToSubmissionForm(submission.id)"
+                :data-testid='`mod-submission-list-item-${index + 1}`'
+                class="grid grid-cols-subgrid col-span-5 bg-gray-200">
```

Comment:
I'm sure you're going to grumble at this one 😆, but since it's clickable we should add in the pointer curse option and maybe through in a nice hover effect ✨